server/routes/linkTokens.js

```diff
@@ -49,7 +49,7 @@ router.post(
         access_token: accessToken,
       };
       // If user has entered a redirect uri in the .env file
-      if (redirect_uri.length > 1) {
+      if (redirect_uri.indexOf('http') === 0) {
```

Comment:
I'm wondering if we should error when the redirect_uri is formatted incorrectly so the user knows what went wrong and how to fix it.

Reply:
will make a note to do this in the future